extra/resources/SysInfo

```diff
@@ -149,11 +149,7 @@ UpdateStat() {
     name=$1; shift
     value="$*"
     printf "%s:\t%s\n" "$name" "$value"
-    if [ "$__OCF_ACTION" = "start" ] ; then
-        ${HA_SBIN_DIR}/attrd_updater ${OCF_RESKEY_delay} -S status -n $name -B "$value"
-    else 
-        ${HA_SBIN_DIR}/attrd_updater ${OCF_RESKEY_delay} -S status -n $name -v "$value"
-    fi
+    ${HA_SBIN_DIR}/attrd_updater ${OCF_RESKEY_delay} -S status -n $name -B "$value"
```

Comment:
The -B to -v mapping happens implicitly (as with -U) if we just allow it in attrd_updater (which the
first patch does).
So I suggest just to leave out the 2nd one. I'll remove it from the branch ...